index.html

```diff
@@ -0,0 +1,53 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Portfolio setup and mobile first</title>
+    <link rel="stylesheet" href="index.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Poppins&display=swap" rel="stylesheet">
+</head>
+
+<body>
+    <header class="header1">
+
+        <h3 class="logo">My Logo</h3>
+        <a href="#"><img src="./img/hamburger.png" alt="hamburger_image"></a>
+
```

Comment:
- Great job wrapping your logo and hamburger menu in a `<header>` tag. ✔️ 

- I think your Logo should be a link. You can do this by wrapping `My Logo` around an `<a>` tag as this is for three reasons: convention, semantics, and user experience(UX). [Check this link](https://ux.stackexchange.com/questions/81727/why-is-it-standard-for-a-website-logo-to-navigate-to-the-home-page) for more info.

- I believe it will be better for your `hamburger menu` to be wrapped around a `<nav>`  inorder to aid user experience for mobile devices. 😉 

You can simply re factor your code to look something like this 👇🏽 
```
   <header class="header1">
        <h3 class="logo"><a href="#">My Logo</a></h3>
        <nav>
             <img src="./img/hamburger.png" alt="hamburger_image">
         </nav>
   </header>

```

What do you think about this suggestion?? 🤔